src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -186,6 +186,12 @@ public Mono<Void> replaceFilterWithScript(UUID id, String userId) {
 
     public Mono<Void> deleteElement(UUID id, String userId) {
         return directoryService.deleteElement(id, userId)
-            .doOnSuccess(e -> directoryService.deleteDirectoryElement(id, userId).subscribe());
+                .doOnSuccess(e -> directoryService.deleteDirectoryElement(id, userId).subscribe())
+                // FIXME dirty fix to ignore errors and still delete the elements in the directory-server. To delete when handled properly.
+                .onErrorResume(e -> {
+                    directoryService.deleteDirectoryElement(id, userId).subscribe();
+                    e.printStackTrace();
```

Comment:
In stead of e.printStackTrace(), one could use the Mono .log(), no ?

Reply:
used a classic logger instead